packaging/debian/datadog-agent.init

```diff
@@ -176,6 +177,11 @@ case "$1" in
         DOGSTATSD_RETURN=$?
         su $AGENTUSER -c "$FORWARDERPATH info"
         FORWARDER_RETURN=$?
+        TRACEAGENT_RETURN=0
+        if [ -x $TRACEAGENTPATH ];
```

Comment:
applies to the 2 other init files as well: I don't think we need to check that the trace agent binary exists and is executable, these init files are only used by the packaged Linux agent and it'll always ship the trace agent going forward, with correct exec permissions. Unless I'm missing something.